scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -122,6 +138,17 @@ class ScanCode(config: ScannerConfiguration) : LocalScanner(config) {
     override val scannerVersion = "2.9.7"
     override val resultFileExt = "json"
 
+    private val scanCodeConfiguration = config.scanner?.get("ScanCode") ?: emptyMap()
+
+    private val configurationOptions = scanCodeConfiguration["commandLine"]?.split(" ")
+            ?: DEFAULT_CONFIGURATION_OPTIONS
+    private val nonConfigurationOptions = scanCodeConfiguration["commandLineNonMod"]?.split(" ")
```

Comment:
Any reason to not call this `commandLineNonConfig` in the configuration?

Reply:
Done, also adapted the documentation.